README.md

```diff
@@ -2,10 +2,10 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.1.
```

Comment:
Emi, check if we have to update version here (9.0.1 -> 11.?)

Reply:
The thing is, the original codebase was generated using that version (9.0.1), but in order to serve the app with the new Angular version, you will have to update to 11. I'm not sure what to do here 😛